x-pack/plugins/monitoring/server/telemetry_collection/get_all_stats.ts

```diff
@@ -21,15 +21,14 @@ type PromiseReturnType<T extends (...args: any[]) => any> = ReturnType<T> extend
 export interface CustomContext {
   maxBucketSize: number;
 }
-
 /**
  * Get statistics for all products joined by Elasticsearch cluster.
  * Returns the array of clusters joined with the Kibana and Logstash instances.
  *
  */
 export const getAllStats: StatsGetter<CustomContext> = async (
   clustersDetails,
-  { callCluster, start, end },
+  { callCluster, start, end, esClient },
```

Comment:
`esClient` do we need to get it even though it's not used inside the function?

Reply:
StatsCollectionConfig requires the `esClient`. If I make `esClient` an optional type then we need to check that it's defined everywhere it's used otherwise typescript complains.